content/applications/hr/payroll/payslips.rst

```diff
@@ -221,144 +240,100 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
+environment, it is **not** possible to modify the :guilabel:`Company` from the form. The batch must
+be created while in the database for the desired company.
 
-Lastly, in the :guilabel:`Company` field, select the company these payslips are written against.
-
-When all the information on the payslip batch form is correct, click the :guilabel:`Save` button to
-save the information. To delete the entry, click the :guilabel:`Discard` button.
-
-To create the payslips for the newly-created batch, click the :guilabel:`Generate Payslips` button
-at the top of the form.
-
-When clicked, a pop-up window appears showing all the payslips that will be created. To remove any
-individual payslips, click the black :guilabel:`✖ (x mark)` icon at the far right of the payslip
-line.
-
-If a specific :guilabel:`Salary Structure` or :guilabel:`Department` needs to be specified for the
-batch, select them from the corresponding drop-down menus.
-
-Click the :guilabel:`Generate` button at the bottom of the pop-up window to create the payslips for
-the batch.
-
-.. image:: payslips/generate-payslips.png
+.. image:: payslips/new-batch-details.png
    :align: center
-   :alt: Generate payslips for the new batch.
+   :alt: Enter the details for the new batch.
 
-.. note::
-   On the :guilabel:`Generate Payslips` pop-up window, selecting a :guilabel:`Department` and/or
-   :guilabel:`Salary Structure` only displays payslips that apply to those specifically-selected
-   parameters.
+When all the information on the payslip batch form is correct, click the :guilabel:`Add Payslips`
+button at the top-left of the page. A pop-up window appears with all the available payslips that can
+be added to the batch.
 
-If there are any errors or issues preventing the payslips from being generated, an error message
-appears in the top-right section. This error box disappears on its own after several seconds, or the
-:guilabel:`✖ (x mark)` icon can be clicked to close the alert.
+Click the checkbox to the left of each payslip to select it. When all desired payslips have been
+selected, click the :guilabel:`Select` button. The pop-up window closes, and the payslips have been
+added to the batch.
 
-To remedy the issue, make any necessary modifications (e.g. removing any payslip lines that cannot
-be processed), then click the :guilabel:`Generate` button again.
+.. _payroll/batch-process:
 
-Once the payslips have been successfully generated, the screen returns to the payslip batch form.
+Process a batch
+---------------
 
-From here, click the :guilabel:`Generate Draft Entry` button to change the payslips' status from
-:guilabel:`Draft` to :guilabel:`Done`.
+Click on an individual batch to view the details for that batch on a separate page. On this batch
+detail page, different options (buttons) appear at the top, depending on the status of the batch:
+
+- :guilabel:`New`: batches without any payslips added to them have a status of :guilabel:`New`.
```

Comment:
is this to represent the New button or New status?

Reply:
I totally redid this section so it is WAY more clear! I can see how confusing this was to read now that I am looking at it a third time!